modules/ocf_prometheus/files/rules.d/mirror.rules.yml

```diff
@@ -1,3 +1,6 @@
+# TODO: add an annotation explaining that these alerts can be investigated by
+# reading {{ $labels.project }}.service and {{ $labels.project }}-health.service
+# on fallingrocks
```

Comment:
How long would this take to actually add? Could this be done instead of adding it as a `TODO`?

It sounds to me like this is a pretty simple change, so that why I'm asking this, but let me know if this is more complicated or if it's not feasible to do for some reason.

Reply:
I wanted to give myself more time to think about a convention for alert annotations, but I should do this now because I'll probably forget :P